vowpalwabbit/parser.cc

```diff
@@ -737,6 +737,10 @@ void setup_example(vw& all, example* ae)
     ae->total_sum_feat_sq += fs.sum_feat_sq;
   }
 
+  // Reset any previously counted interacted features.
```

Comment:
should this cleanup be done when we are releasing the example back to the pool during example cleanup?

Reply:
That's a good idea, I'll do it there too